tutorials/networking/high_level_multiplayer.rst

```diff
@@ -371,9 +372,151 @@ have loaded the game scene.
         players.clear()
         server_disconnected.emit()
 
+ .. code-tab:: csharp C#
+    using Godot;
+    using Godot.Collections;
+
+    public partial class Lobby : Node
+    {
+        // These signals can be connected to by a UI lobby scene or the game scene.
+        [Signal]
+        public delegate void PlayerConnectedEventHandler(int peerId, Dictionary<string, string> playerInfo);
+
+        [Signal]
+        public delegate void PlayerDisconnectedEventHandler(int peerId);
+
+        [Signal]
+        public delegate void ServerDisconnectedEventHandler();
+
+        private const int _port = 7000;
+        private const string _defaultServerIp = "127.0.0.1";
+        private const int _maxConnections = 20;
+
+        // This will contain player info for every player, with the keys being each player's unique IDs.
+        private readonly Dictionary<long, Dictionary<string, string>> _players = new();
```

Comment:
I don't think this collection is used with Godot API, in that case the recommendation is to use a System collection (see the [_C# collections_](https://docs.godotengine.org/en/stable/tutorials/scripting/c_sharp/c_sharp_collections.html#choose-a-collection) documentation page).

Reply:
I'm confused. Unless I'm mistaken it looks like you can use `Godot.Collections.Dictionary`, right?